controls/radtreelistview/radtreeliestview-getting-started.md

```diff
@@ -78,36 +56,21 @@ In order to use the __RadTreeListView__ in XAML, you have to declare the followi
 {{endregion}}
 
 
+## Prepare Sample Data
 
-#### __C#__
-
-{{region radtreeliestview-getting-started_2}}
-
-	RadTreeListView radTreeListView = new RadTreeListView();
-{{endregion}}
-
-
-
-#### __VB.NET__
-
-{{region radtreeliestview-getting-started_3}}
-
-	Dim radTreeListView As New RadTreeListView()
-{{endregion}}
-
-
-
-## Prepare a Sample Data
+For this example we will use a collection that represents the contents of a warehouse. It will contain __WarehouseItem__ objects. Each __WarehouseItem__ will have a __Name__, __Count__ and a collection of __WarehouseItem__ objects. Here is the class definition.

```

Comment:
Add an introductory sentence to the section - what does it demonstrate, then proceed with the example. Avoid "we" when working on a tutorial and provide steps to the user - "For this example, create a collection...".